module/mempool/herocache/backdata/heropool/pool.go

```diff
@@ -149,13 +148,13 @@ func (p Pool) All() []PoolEntity {
 	return all
 }
 
-// Head returns the head of used items. Assuming no ejection happened and pool never goes beyond limit, Head returns
+// Head returns the head of states[stateUsed] items. Assuming no ejection happened and pool never goes beyond limit, Head returns
```

Comment:
There is no need to change the comments in this way; the old version of the godocs is preferred, please refactor it across all changes.

Reply:
ok, will do